workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-css-4/client-extension.yaml

```diff
@@ -0,0 +1,13 @@
+assemble:
+    - from: build/buildTheme/img
+      into: static/img
+    - from: build/buildTheme/images
+      into: static/images
```

Comment:
Are `img` and `images` two different folders? Maybe we could rename them :thinking:  since these words have same meaning

Reply:
cool :+1: